<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GantryWindow</class>
 <widget class="QMainWindow" name="GantryWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1440</width>
    <height>851</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="ItemDisplayWindow" name="itemDisplay" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>441</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>280</y>
      <width>171</width>
      <height>362</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="homeButton">
       <property name="text">
        <string>Home</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="loadCalButton">
       <property name="text">
        <string>Load Calibration</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="positionButton">
       <property name="text">
        <string>Poll Position</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="positionInput"/>
     </item>
     <item>
      <widget class="QPushButton" name="moveButton">
       <property name="text">
        <string>Custom Position</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="arduinoInput"/>
     </item>
     <item>
      <widget class="QPushButton" name="arduinoButton">
       <property name="text">
        <string>Send to Arduino</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QVTKWidget" name="qvtkWidget" native="true">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>280</y>
      <width>631</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="statusDisplay">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>280</y>
      <width>271</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>20</y>
      <width>181</width>
      <height>103</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QDoubleSpinBox" name="cal_x_spin"/>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="cal_y_spin"/>
     </item>
     <item>
      <widget class="QPushButton" name="clearObjectsButton">
       <property name="text">
        <string>Clear Objects</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>20</y>
      <width>171</width>
      <height>251</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPlainTextEdit" name="x_pte">
       <property name="font">
        <font>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="y_pte"/>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="z_pte"/>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="j4_pte"/>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="j5_pte"/>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="j6_pte"/>
     </item>
    </layout>
   </widget>
   <widget class="ChartView" name="chartView" native="true">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>20</y>
      <width>271</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>130</y>
      <width>180</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="prevItemButton">
         <property name="text">
          <string>Previous</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="nextItemButton">
         <property name="text">
          <string>Next</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="scanButton">
       <property name="text">
        <string>Scan</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="centerItemButton">
       <property name="text">
        <string>Center</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1440</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuELEC_490_Group_19_Gantry_Control">
    <property name="title">
     <string>ELEC 490 Group 19 : Gantry Control</string>
    </property>
   </widget>
   <addaction name="menuELEC_490_Group_19_Gantry_Control"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
  <customwidget>
   <class>ItemDisplayWindow</class>
   <extends>QWidget</extends>
   <header>itemdisplaywindow.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ChartView</class>
   <extends>QWidget</extends>
   <header>chartview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
